arch/arc/Kconfig

```diff
@@ -85,13 +85,35 @@ config ARC_FIRQ
 	  If FIRQ is disabled, the handle of interrupts with highest priority
 	  will be same with other interrupts.
 
-config	ARC_STACK_CHECKING
+config 	ARC_HAS_STACK_CHECKING
+	bool "ARC has STACK_CHECKING"
+	default y
+	help
+	  ARC is configured with STACK_CHECKING which is a mechanism for
+	  checking stack accesses and raising an exception when a stack
+	  overflow or underflow is detected.
+
+config  ARC_STACK_CHECKING
+	bool
+	# Omit prompt to signify "hidden" option
```

Comment:
Maybe these comments could be skipped. They get copied around everywhere. :)

Reply:
already fixed